examples/sync-localhost.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
remove this file, unless you used it

Reply:
add the new fields destination, prefix and stripPrefix to config-sync.json and also in the examples/README.md.

Reply:
I have updated examples/config-sync.json and README. I am not quite sure if my description suffices.